src/tests/test_example_diffs.rs

```diff
@@ -831,6 +831,33 @@ src/align.rs
         );
     }
 
+    #[test]
```

Comment:
Thanks a lot for writing this test, this is extremely useful!

Can we change the name of the test to `test_color_only_output_is_in_one_to_one_correspondence_with_input`?

Also, how about adding a couple more tests, like

```rust
    #[test]
    fn test_color_only_output_is_in_one_to_one_correspondence_with_input_1() {
        let config = integration_test_utils::make_config_from_args(&["--color-only", "true"]);
        let output = integration_test_utils::run_delta(GIT_DIFF_SINGLE_HUNK, &config);

        let output = strip_ansi_codes(&output);
        let output_lines: Vec<&str> = output.split('\n').collect();
        let input_lines: Vec<&str> = GIT_DIFF_SINGLE_HUNK.split('\n').collect();

        assert_eq!(input_lines.len(), output_lines.len());

        for n in 0..input_lines.len() {
            assert_eq!(input_lines[n], output_lines[n]);
        }
    }

    #[test]
    fn test_color_only_output_is_in_one_to_one_correspondence_with_input_2() {
        let config = integration_test_utils::make_config_from_args(&[
            "--color-only",
            "true",
            "--file-style",
            "blue",
            "--commit-style",
            "red",
            "--hunk-header-style",
            "syntax",
            "--hunk-header-decoration-style",
            "box",
        ]);
        let output = integration_test_utils::run_delta(GIT_DIFF_SINGLE_HUNK, &config);

        let output = strip_ansi_codes(&output);
        let output_lines: Vec<&str> = output.split('\n').collect();
        let input_lines: Vec<&str> = GIT_DIFF_SINGLE_HUNK.split('\n').collect();

        assert_eq!(input_lines.len(), output_lines.len());

        for n in 0..input_lines.len() {
            assert_eq!(input_lines[n], output_lines[n]);
        }
    }

```

Reply:
Of cource! Yes, the name is more clear. Thank you for pointing out.

Reply:
Added here 4fd75f1